pkgs/by-name/su/substudy/package.nix

```
@@ -0,0 +1,50 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+
+  ffmpeg,
+  makeWrapper,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "substudy";
+  version = "0.6.10";
+
+  src = fetchFromGitHub {
+    owner = "emk";
+    repo = "subtitles-rs";
+    rev = "${pname}_v${version}";
+    hash = "sha256-ACYbSQKaOJ2hS8NbOAppfKo+Mk3CKg0OAwb56AH42Zs=";
+  };
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+    outputHashes = {
+      "async-openai-0.19.1" = "sha256-UrWSZW3LxMZfinmQAjouhqTrhVhHkjgz9EzrZxR0qG4=";
+    };
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  nativeCheckInputs = [ ffmpeg ];
+
+  cargoBuildFlags = [ "-p substudy" ];
+
+  preCheck = ''
+    export HOME=(mktemp -=d)
```

This doesn't do anything?

```suggestion
    export HOME=$(mktemp -d)
```

or 
```suggestion
```

---

That's to make sure the [`test_ai_request_static`](https://github.com/emk/subtitles-rs/blob/d11ece81ec2c276e2d78798009f628d565fdb1cf/substudy/src/ai.rs#L175) test has access to the [cache](https://github.com/emk/subtitles-rs/blob/d11ece81ec2c276e2d78798009f628d565fdb1cf/substudy/src/ai.rs#L102) when necessary. While packaging the program, I found this test to be flaky (it will fail sometimes).

No idea what happened with the syntax, though... Probably a couple of typos.